notes/mfip-orientation.vbs

```diff
@@ -0,0 +1,957 @@
+		cancel_without_confirmation
+
+	    'Handling for error messaging (in the case of mandatory fields or fields requiring a specific format)-----------------------------------
+		Call validate_MAXIS_case_number(err_msg, "*")																	'case number is mandatory here
+		IF worker_signature = ""           THEN err_msg = err_msg & vbNewLine & "* You must sign your case note!"       'worker_signature is usually also a mandatory field
+
+		If ButtonPressed = mfip_orientation_instructions_btn Then				'This button will open the instructions and then reshow the dialog
+			err_msg = "LOOP"
+			run "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe https://hennepin.sharepoint.com/:w:/r/teams/hs-economic-supports-hub/BlueZone_Script_Instructions/NOTES/NOTES%20-%20INTERVIEW%20-%20MFIP%20ORIENTATION.docx"
+		End If
+	    'If the error message isn't blank or if the instructions button wasn't pressed, it'll pop up a message telling you what to do!
+		IF err_msg <> "" and err_msg <> "LOOP" THEN MsgBox "*** NOTICE!!! ***" & vbNewLine & err_msg & vbNewLine & vbNewLine & "Please resolve for the script to continue."     '
+	LOOP UNTIL err_msg = ""     'It only exits the loop when all mandatory fields are resolved!
+	call check_for_password(are_we_passworded_out)		'ensuring we did not become passworded out while the dialog was up
+Loop until are_we_passworded_out = False
+
+Call back_to_SELF
+EMWriteScreen MAXIS_case_number, 18, 43
+EMReadScreen MX_region, 10, 22, 48
+MX_region = trim(MX_region)
+If MX_region = "INQUIRY DB" Then
+	continue_in_inquiry = MsgBox("You have started this script run in INQUIRY." & vbNewLine & vbNewLine & "The script cannot complete a CASE:NOTE when run in inquiry. The functionality is limited when run in inquiry. " & vbNewLine & vbNewLine & "Would you like to continue in INQUIRY?", vbQuestion + vbYesNo, "Continue in INQUIRY")
+	If continue_in_inquiry = vbNo Then Call script_end_procedure("~PT MFIP Orientation Script cancelled as it was run in inquiry.")
+End If
+
+'QUESTION - should we check to ensure MFIP is active or pending, or at least Cash is pending?
+
+'We are not using the function here because we do not need to select the members to look at.
+'The function has the ability to select which members of the household are caregivers in the next dialog.
+Call navigate_to_MAXIS_screen_review_PRIV("STAT", "MEMB", is_this_priv)			'navigating to stat memb to gather the list of all ref number on the case
+If is_this_priv = True Then Call script_end_procedure("It appears that this case is PRIVILEGED and you do not currently have access to it. Check the Case Number and, if needed, request access. Run the script agian once you confirm you have access to the case. The script will now end.")
+EMWriteScreen "01", 20, 76
+transmit
+
+EMReadScreen case_pw_county, 2, 21, 19											'check to see if the case is in another county
```

Comment:
MEMB reads the case_pw_county at 2, 29, 23

Reply:
I think this is 21, 23 - that is what I changed it to.